### PR TITLE
Remove check that prevents fuzzing below minimum size

### PR DESCRIPTION
**Pull request by Alan-Jowett**
## Description

Remove check in fuzzer to permit varying request and reply length.

## Testing

CI/CD

## Documentation

CI/CD

## Installation

No.

### CODE REVIEW

**Review by dthaler — DISMISSED** [review]

File: libs/execution_context/ebpf_core.c [hunk 1]
```diff
@@ -999,21 +999,52 @@ _ebpf_core_protocol_program_test_run(
...
+    retval = ebpf_safe_size_t_subtract(
+        context_size_in, EBPF_OFFSET_OF(ebpf_operation_program_test_run_request_t, data), &context_size_in);
     if (retval != EBPF_SUCCESS) {
+        // Request isn't big enough to contain the header.
         goto Done;
     }
 
-    if (data_in_end > request->header.length) {
-        retval = EBPF_INVALID_ARGUMENT;
+    // Subtract the dat size.
```
Comment (dthaler) [inline libs/execution_context/ebpf_core.c:1019]:
```suggestion
    // Subtract the data size.
```

**Review by dthaler — DISMISSED** [review]
Ok except for a typo

**Review by gtrevi — CHANGES_REQUESTED** [review]
We should address default behavior in all `switch` statements.

**Review by saxena-anurag — DISMISSED** [review]

**Review by dthaler — DISMISSED** [review]
Re-approving after typo fix

**Review by dthaler — APPROVED** [review]

**Review by gtrevi — APPROVED** [review]

**Pull request merged**